core/src/main/scala/sbt/librarymanagement/LibraryManagementSyntax.scala

```diff
@@ -0,0 +1,24 @@
+package sbt.librarymanagement
+
+abstract class LibraryManagementSyntax extends DependencyBuilders {
```

Comment:
ok. I guess I'll turn them into traits and put comments to remind future developers to use final val.